README.md

```diff
@@ -25,6 +25,12 @@ in this [blog post](https://blog.nlnetlabs.nl/testing-the-waters-with-krill/).
 
 # Changelog
 
+## 0.14.4 'A Flock of Krill'
+
+This release fixes an issue found in 0.14.0 through 0.14.3:
+
+- KVX gets stuck on locks #1171.
```

Comment:
I rephrased #1171 to: "KVX gets stuck if lockfiles are not deleted #1171"

Perhaps that's more clear?